app/models/queue_item.rb

```diff
@@ -0,0 +1,18 @@
+class QueueItem < ActiveRecord::Base
+
+  validates_presence_of :user_id, :video_id
+  belongs_to :user
+  belongs_to :video
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+
+  def category_name
+    video.category.title
```

Comment:
Since you delegated `category` to `video`, there is no need for the `video.` portion.